src/api/manager/job/syncInventory.ts

```diff
@@ -67,26 +73,30 @@ export async function runSyncInventory() {
       // If source was not found in response from API, always mark it as gone
       return { ...inventorySource, status: 'gone' } satisfies WithId<Source>;
     }
-    // Keep all old fields from the inventory source (name, tags, id, audio_stream etc), but update the status
+    // Keep all old fields from the inventory source (name, tags, id, audio_stream etc), but update the status and set the lastConnected to the current date
     return {
       ...inventorySource,
-      status: apiSource.status
+      status: apiSource.status,
+      lastConnected: new Date()
```

Comment:
I just learned ( yesterday ) that the API now actually stores _some_ data about previously connected sources ( they used to just disappear...  ) so we have check the status here and only update lastConnected _if_ `status !== "gone"`.